includes/.bashrc-liferay

```diff
@@ -222,6 +222,7 @@ function openCommitsIn {
     echo "Ticket not found, please use 'openCommitsIn <XXXXX-0000>' format and verify your ticket exists in this branch. e.g. 'openCommitsIn LOOP-1234'"
   }
 
+  cwd=`pwd`
```

Comment:
cwd should be declared as a local variable so it's destroyed upon function exit.